src/cmake_config.h.in

```diff
@@ -40,3 +40,4 @@
 #cmakedefine01 CURSES_HAVE_NCURSESW_CURSES_H
 #cmakedefine01 BUILD_UNITTESTS
 #cmakedefine01 BUILD_BENCHMARKS
+#cmakedefine01 USE_SDL2
```

Comment:
I'm not sure this works reliably when the variable is defined in `lib/irr`

Reply:
I did a quick test with

```cpp
errorstream << "USE_SDL2 = " << USE_SDL2 << std::endl;
```

and it worked as expected. But I have no idea whether it's permitted/reliable.

Reply:
I guess it's okay as a workaround until we switch to SDL